src/depends.py

```diff
@@ -1,52 +1,199 @@
-#! python
+"""
+Utility functions to check the availability of dependencies
+and suggest how it may be installed
+"""
 
 import sys
-import os
-import pyelliptic.openssl
 
 # Only really old versions of Python don't have sys.hexversion. We don't
 # support them. The logging module was introduced in Python 2.3
 if not hasattr(sys, 'hexversion') or sys.hexversion < 0x20300F0:
```

Comment:
During testing android porting, it turned out that it needs at least 2.7.4. 2.7.3 has a bug in struct.unpack which breaks PyBitmessage. I tested it personally with 2.7.3 and 2.7.4 today. 

Historical references: https://bitmessage.org/forum/index.php?topic=4081.0

Reply:
2.3 is checked here to exit before `import logging`. I think it makes sense because 2.7.4 is checked below.